css/employees.css

```diff
@@ -0,0 +1,411 @@
+@import url("common.css");
+

```

Comment:
i still see there are some duplicate styles like `.roles-selection-menu`
please remove all duplicate styles and move it to common.css file